client/src/components/Header.js

```diff
@@ -119,7 +119,7 @@ export const Header = ({
                 </div>
                 <div className={styles['action-container']}>
                   <Button onClick={handleOnClickSaveAll} buttonStyle="actionbar-save" text="Save all"></Button>
-                  <Button onClick={handleOnClickValidateAll} buttonStyle="actionbar-validate" text="Validate all"></Button>
+                  <Button onClick={handleOnClickValidateAll} buttonStyle="actionbar-validate" text="Spellcheck"></Button>
```

Comment:
Could we change "Save All" to Save Document and then "Spellcheck" can go to Spellcheck Document?

Reply:
I think that appending Document would make the label unnecessarily wide, without adding any information since "Saving document" already appears in the loading message.
How about simply making it "Save" and "Spellcheck"? I think it would be more elegant.

Reply:
Sounds good to me